plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -258,7 +258,7 @@
 
     private String _updateHostPasswdPath;
 
-    private int _dom0MinMem;
+    private long _dom0MinMem;
```

Comment:
Agree, I think we should always reserve at least 1GB to keep a system alive. Admins can always set it so something lower.